### DISCUSSION TITLE
现在apollo-client支持@ConfigProperties的自动更新嘛

### DISCUSSION THREAD

**Discussion by Pwenlee** [apolloconfig/apollo-java#23] [General]
现在apollo-client支持@ConfigProperties的自动更新嘛

**Comment by nobodyiam** [comment]
see [3.2.2.3 ConfigurationProperties使用方式](https://www.apolloconfig.com/#/zh/usage/java-sdk-user-guide?id=_3223-configurationproperties%e4%bd%bf%e7%94%a8%e6%96%b9%e5%bc%8f)

**Comment by Pwenlee** [comment]
Hello,this is AutoReply, I have received your E-mail, now it is not convenient to reply, please forgive!I will deal with it as soon as possible.
